python/aitemplate/backend/cuda/target_def.py

```diff
@@ -136,10 +136,15 @@ def get_host_compiler_options(self) -> List[str]:
         return self._build_gnu_host_compiler_options()
 
     def _build_nvcc_compiler_options(self) -> List[str]:
+        code = [f"sm_{self._arch}", f"compute_{self._arch}"]
+        code = ",".join(
```

Comment:
Nit: Instead of joining it here, could you simply add `f"lto_{self._arch}"` to the `code` list if required, and then use `','.join(code)` directly in the L147 below?